x-pack/plugins/security_solution/public/common/components/text_field_value/index.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiToolTip } from '@elastic/eui';
+import React from 'react';
+
+const trimTextOverflow = (text: string, maxLength?: number) => {
+  if (maxLength !== undefined && text.length > maxLength) {
+    return `${text.substr(0, maxLength)}...`;
+  } else {
+    return text;
+  }
+};
+
+interface Props {
+  fieldName: string;
+  value: string;
+  maxLength?: number;
+  className?: string;
+}
+
+export const TextFieldValue = ({ fieldName, value, maxLength, className }: Props) => {
+  return (
+    <EuiToolTip
```

Comment:
Is it by design that the tooltip will aways show even if the value is not truncated?

Reply:
Yes. By my design :)

The thing is we can not detect CSS truncation for one, secondly that's the behaviour of formatted date - it always shows tooltips, so I decided it would be consistent if we show tooltip for all important values.

Reply:
Sorry for the delayed response @efreeti 

I would still propose that you have a quick discussion/review with @bfishel on this. Displaying a tooltip evertime even where there is no additional value from it may not be the desired UX. 

Also, I would not equate this to the Component we use for dates - that one has a different purpose. The date component that includes a tooltip is primarily used to show relative time ++ formatted local time ++ raw date value (usually an ISO8601 string)